frontend/middleware.js

```diff
@@ -59,21 +33,17 @@ export function middleware(req) {
         </style>
       </head>
       <body>
-        <h1>Error 451: Unavailable For Legal Reasons</h1>
-        <p>This Site is not available to residents of Belarus, the Central African Republic, The Democratic Republic of Congo, the Democratic People's Republic of Korea, the Crimea, Donetsk People's Republic, and Luhansk People's Republic regions of Ukraine, Cuba, Iran, Libya, Somalia, Sudan, South Sudan, Syria, the USA, the United Kingdom, Yemen, Zimbabwe and any other jurisdiction in which accessing or using the Site is prohibited (the "Prohibited Jurisdictions").</p>
+        <h1>The airdrop claim period has ended</h1>
+        <p>To stay in touch with future Pyth community initiatives head over to our <a href="https://discord.gg/invite/PythNetwork">Discord</a>  </p>
```

Comment:
can we change the styling of the <a> tag? right now its the default blue which doesn't go well with our bg